src/styles/calendar.scss

```diff
@@ -0,0 +1,58 @@
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  width: calc($day-size * 7 + $gap-size * 6);
```

Comment:
Ensure that the variable `$day-size` is defined somewhere in your SCSS files. Without this definition, the calculation for the calendar width will not work.